doc/nrf/external_comp/nrf_cloud.rst

```diff
@@ -34,6 +34,46 @@ You can use the services offered by nRF Cloud in the following scenarios:
 * Device connected to nRF Cloud over REST, interacting using the `nRF Cloud REST API`_.
 * Device connected to a customer cloud service in a suitable manner. The services can be used from the customer cloud service that communicates over REST to the nRF Cloud REST API in a proxy configuration.
 
+Choosing a device ID format: prefix-IMEI, device UUID, or other
+***************************************************************
+
+nRF Cloud supports a variety of device ID formats:
+
+* A prefix plus IMEI
+  This is reserved for Nordic development kits or evaluation platforms when the prefix is ``nrf-``.
+* The device UUID
+  This is a guaranteed unique ID stored in the modem.
+* Compile-time
+  A fixed device ID compiled into the firmware.
+* Run-time
+  A device ID generated by the application in an application-specific way.
+* HW-ID
+  A device ID generated from one of a selection of hardware IDs.
+
+Offical Nordic development kits such as the nRF9160DK have a sticker on the back indicating the device's IMEI and PIN.
+They are preprovisioned with JITP certificates so they can be added easily to nRF Cloud using the device ID ``nrf-IMEI`` and PIN.
+Customers using custom hardware cannot use ``nrf-<IMEI>`` style device ids.
+Names that start with ``nrf-`` and their PIN or HWID are checked against a database of genuine Nordic development kits and Thingy:91s.
+If not found in the database, the device cannot connect.
+PINs/HWIDs are not needed for customers to connect custom devices to nRF Cloud.
+
+The device_credentials_installer.py script supports either method.
+See these command line options:
+--id_str ID_STR       Device ID to use instead of UUID. Will be a prefix if used with --id_imei (default: )
+--id_imei             Use IMEI for device ID instead of UUID. Add a prefix with --id_str (default: False)
+-d, --delete          bool: Delete sectag from modem first (default: False)
+
+The device_device_credentials_installer.py automatically connects over USB to a device and installs the certificates in the modem -- the customer does not need to use nRF Connect for Desktop’s Cellular Monitor.
+You need to install and run firmware on the device beforehand which accepts AT commands over the USB, like at_client.
+
+You must build you firmware to match this – see:
```

Comment:
```suggestion
You must build your firmware to match this – see:
```

Reply:
i think this statement should be moved up a bit... and maybe modified to something like:

"Your firmware's device ID configuration _(link to NRF_CLOUD_CLIENT_ID_SRC)_ must align with the options used when executing device_credentials_installer.py "